src/NServiceBus.Hosting.Windows/packages.config

```diff
@@ -1,7 +1,7 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="CommonServiceLocator" version="1.0" targetFramework="net45" />
-  <package id="Fody" version="1.26.1" targetFramework="net45" developmentDependency="true" />
+  <package id="CommonServiceLocator" version="1.3" targetFramework="net45" />
```

Comment:
Was it needed to upgrade the service locator? (just afraid it might break things since we rely on an ancient version of TopShelf)